core/src/main/java/net/adoptopenjdk/icedteaweb/client/parts/splashscreen/SplashUtils.java

```diff
@@ -169,52 +183,42 @@ static SplashErrorPanel getErrorSplashScreen(int width, int height, SplashUtils.
     }
 
     /**
+     * Returns a splash or null if splash is suppressed by {@link IcedTeaWebConstants.ICEDTEA_WEB_SPLASH} environment variable
+     *
      * @param width
      * @param height
      * @param splashReason
      * @param loadingException
      * @param isError
      */
-    public static SplashPanel getSplashScreen(int width, int height, SplashUtils.SplashReason splashReason, Throwable loadingException, boolean isError) {
+    public static SplashPanel getSplashScreen(final int width, final int height, final SplashUtils.SplashReason splashReason, final Throwable loadingException, final boolean isError) {
+        SplashPanel splashPanel;
         String splashEnvironmentVar = null;
-        String pluginSplashEnvironmentVar = null;
+
```

Comment:
If you extract `String splashEnvironmentVar = null` togehter with the try block into a new method you can make the splashEnvironmentVar final

Reply:
extracted to method